src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -119,12 +119,13 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
 
   private def writeAuthenticationFile(workflow: WorkflowDescriptor) = authenticated { connection =>
     val path = GoogleCloudStoragePath(gcsAuthFilePath(workflow))
+    val log = workflowLogger(workflow)
 
     generateAuthJson(jesConf.dockerCredentials, getGcsAuthInformation(workflow)) foreach { content =>
       def upload(prev: Option[Unit]) = connection.storage.uploadJson(path, content)
 
-      logger.info(s"Creating authentication file for workflow ${workflow.id} at \n ${path.toString}")
-      withRetry(upload, s"${makeTag(workflow)} Exception occurred while uploading auth file to $path")
+      log.info(s"Creating authentication file for workflow ${workflow.id} at \n ${path.toString}")
+      withRetry(upload, s"${log.makeTag} Exception occurred while uploading auth file to $path")
```

Comment:
In this case, it uses the lazy logger, so it'll end up in the `cromwell.<date>.log` file but not the individual workflow's log.  While not terrible, I think I can fix that and make this code cleaner so I don't have to include the `${log.makeTag}`